libs/platform/src/lib/approval-flow/approval-flow.component.html

```diff
@@ -98,6 +98,7 @@
                         class="fdp-approval-flow__graph-container"
                         [class.fdp-approval-flow__graph-container--multiple-root-nodes]="_multipleRootNodes"
                         [class.fdp-approval-flow__graph-container--multiple-final-nodes]="_multipleFinalNodes"
+                        [class.fd-scrollbar]="true"
```

Comment:
use directive instead of css class please.

Reply:
If you think it's not possible in this exact case, you have to import scrollbar's styles from fund-styles to the sass file of the component.